Multiple AssertionErrors occurrences in tests (Debian sid)

Hi,

There are several AssertionError occurrences in tests during the build process in Debian sid. https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1079740

```
self = <optlang.tests.test_glpk_exact_interface.ModelTestCase testMethod=test_objective_expression_includes_constant>

    def test_objective_expression_includes_constant(self):
        objective = self.model.objective
        self.model.objective = self.interface.Objective(objective.expression + 3)
        self.model.update()
>       self.assertEqual((self.model.objective.expression - (objective.expression + 3.)).expand(), 0.)
E       AssertionError: 0 != 0.0

optlang/tests/abstract_test_cases.py:826: AssertionError
```

Following patch should help fix the issue,

```patch
--- a/src/optlang/tests/abstract_test_cases.py
+++ b/src/optlang/tests/abstract_test_cases.py
@@ -22,6 +22,7 @@ import unittest
 from functools import partial
 
 import sympy
+from sympy import simplify
 
 import optlang
 from optlang import interface, symbolics
@@ -823,7 +824,7 @@ class AbstractModelTestCase(unittest.Tes
         objective = self.model.objective
         self.model.objective = self.interface.Objective(objective.expression + 3)
         self.model.update()
-        self.assertEqual((self.model.objective.expression - (objective.expression + 3.)).expand(), 0.)
+        self.assertEqual(simplify(self.model.objective.expression - (objective.expression + 3.)), 0)
 
     def test_is_integer(self):
         model = self.model

```

--- a/CHANGELOG.rst
+++ b/CHANGELOG.rst
@@ -3,6 +3,10 @@
 Next Release
 -----
 
+1.8.3
+-----
+* fix the objective offset test for compatibility with Debian sid
+
 1.8.2
 -----
 * fix the feasibility check in the hybrid solver

--- a/src/optlang/tests/abstract_test_cases.py
+++ b/src/optlang/tests/abstract_test_cases.py
@@ -824,7 +824,8 @@ class AbstractModelTestCase(unittest.TestCase, metaclass=abc.ABCMeta):
         self.model.objective = self.interface.Objective(objective.expression + 3)
         self.model.update()
         self.assertEqual(
-            (self.model.objective.expression - (objective.expression + 3.)).expand().evalf(),
+            float((self.model.objective.expression -
+                  (objective.expression + 3.)).expand().evalf()),
             0.
         )
 

--- a/src/optlang/tests/abstract_test_cases.py
+++ b/src/optlang/tests/abstract_test_cases.py
@@ -823,7 +823,10 @@ class AbstractModelTestCase(unittest.TestCase, metaclass=abc.ABCMeta):
         objective = self.model.objective
         self.model.objective = self.interface.Objective(objective.expression + 3)
         self.model.update()
-        self.assertEqual((self.model.objective.expression - (objective.expression + 3.)).expand(), 0.)
+        self.assertEqual(
+            (self.model.objective.expression - (objective.expression + 3.)).expand().evalf(),
+            0.
+        )
 
     def test_is_integer(self):
         model = self.model

--- a/src/optlang/tests/test_hybrid_interface.py
+++ b/src/optlang/tests/test_hybrid_interface.py
@@ -324,7 +324,11 @@ else:
             self.assertEqual(constraint.name, "test")
             self.assertEqual(constraint.lb, -100)
             self.assertEqual(constraint.ub, None)
-            self.assertEqual((constraint.expression - (0.4 * y + 0.3 * x + 77.0 * z)).expand(), 0.0)
+            self.assertEqual(
+                float((constraint.expression -
+                       (0.4 * y + 0.3 * x + 77.0 * z)).expand().evalf()),
+                0.0
+            )
 
         def test_change_of_objective_is_reflected_in_low_level_solver(self):
             x = Variable('x', lb=-83.3, ub=1324422.)

